### ISSUE TITLE
Long term plan for SmartNora

### ISSUE THREAD

**Issue by nygma2004** [andrei-tatar/node-red-contrib-smartnora#273]
Hi Andrei,

I am a subscriber to your service for a few years now, and I am very happy with it. I was asked to build an integration between a PLC based home automation system and Google Home, and I am planning to proposed Node-Red and SmartNora. This would be a commercial offer and I would obviously offer a long term solution.

 I know this is strange to ask, but still want to make sure that you consider this service stable, and to be around for the next few years?

**Comment by andrei-tatar** [comment]
Hi @nygma2004 ,

I'm happy to hear that!
I have no plan in taking the service offline and I'm also using it on a daily basis for my home automation as all devices from my Google Home are provided via NORA. 
I actually want to extend NORA to support all the traits and devices that have been added to Google Home recently.

> I know this is strange to ask, but still want to make sure that you consider this service stable, and to be around for the next few years?

TLDR: yes.

**Comment by nygma2004** [comment]
@andrei-tatar thanks a lot for your response, and again huge respect for the work that you do. New subscriber would be coming to your way hopefully soon.
